src/components/Button/index.js

```diff
@@ -97,29 +96,10 @@ const styles = {
     },
   },
 };
-const Button = styled(
-  ({
-    intent,
-    isLoading,
-    variant,
-    disabled,
-    type,
-    margin,
-    className,
-    forwardedRef,
-    children,
-    ...rest
-  }) => (
-    <button
-      disabled={disabled}
-      type={type}
-      className={cx('f-Button', className)}
-      ref={forwardedRef}
-      children={isLoading ? <Spinner /> : children}
-      {...rest}
-    />
-  ),
-)`
+const Button = styled('button').attrs(({ className, isLoading, children }) => ({
+  className: cx('f-Button', className),
+  children: isLoading ? <Spinner /> : children,
```

Comment:
I don't think this would work
When a button is loading, we want it to keep the same size, while displaying the loader
That's why we play with elements opacity here
https://github.com/heetch/flamingo/blob/master/packages/react/src/components/Button/index.js#L49-L53

Reply:
You're right, this changes the behavior completely. Will fix and add an extra story to it